code/modules/uplink/uplink_items.dm

```diff
@@ -1745,7 +1745,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			Be warned, that spirits are often capricious or just little assholes. To use, simply speak your question aloud, then begin shaking."
 	item = /obj/item/toy/eightball/haunted
 	cost = 2
-	restricted_roles = list("Curator")
+	restricted_roles = list("Chaplain")
```

Comment:
```suggestion
	restricted_roles = list("Chaplain","Curator")
```

Reply:
Both jobs make sense imo.
I do agree with you Chaplain does need this but I also don't think curator "doesn't" need it. Both seem to fit in their own way.